### ISSUE TITLE
I was trying to import make_forge from mglearn, it threw Import error about load buston. How does load Boston relate with make_forge?

### ISSUE THREAD

**Issue by Mentoni** [amueller/introduction_to_ml_with_python#176]

**Comment by karolinebrito** [comment]
I have the same error. It seems Boston dataset is removed from scikit-learn. Since mglearn lib also calls load_boston, the import of mglearn results in error.

**Comment by karolinebrito** [comment]
I am using Google Colab and used the following:

pip install "scikit-learn<1.2"

When I try to import mglearn, it worked.

**Comment by mfafif0002** [comment]
So sad. I really want to practice by using it

**Comment by wanghedy** [comment]
您的邮件已收到，我将会尽快处理，谢谢。王宇凝

**Comment by amueller** [comment]
Yes, downgrading is currently the only solution, sorry. The book needs to be rewritten with a different dataset.

**Comment by wanghedy** [comment]
您的邮件已收到，我将会尽快处理，谢谢。王宇凝

**Comment by amueller** [comment]
I'll try to do a temporary fix asap.
